bug: Error changing name of namespace

### Description

I'm not sure if there's a rule preventing changes to the first namespace automatically created after ShellHub installation and setup, but I noticed a failure when attempting to rename it.

### Steps to Reproduce

1. Go to Settings > Namespace > Edit Namespace

2. Even not changing anything, try to save.

Error:

![Image](https://github.com/user-attachments/assets/610dafe9-44f7-4320-8557-a9906fd5f387)

![Image](https://github.com/user-attachments/assets/20d4ee12-bfb3-48b2-9d58-fd78a6b70825)

### Expected Behavior

Change the name of the namespace after clicking in Save button.

### Edition

Community

### Version

0.18.3

### Related Logs

```shell
index-Cczh3LB1.js:69 >>>>>>>>>>>>>>>>>>>>>
index-Cczh3LB1.js:14 Error: Cannot read properties of undefined (reading 'connection_announcement')
    at rn (index-Cczh3LB1.js:69:76084)
    at x (SettingNamespace-4IIk-BF_.js:3:3477)
    at se (SettingNamespace-4IIk-BF_.js:3:3729)
gF @ index-Cczh3LB1.js:14
Qm @ index-Cczh3LB1.js:14
(anonymous) @ index-Cczh3LB1.js:14
Promise.catch
Sa @ index-Cczh3LB1.js:14
Sa @ index-Cczh3LB1.js:14
r @ index-Cczh3LB1.js:18
```

### Related Code

_No response_

### Additional Information

_No response_

--- a/ui/src/interfaces/INamespace.ts
+++ b/ui/src/interfaces/INamespace.ts
@@ -29,11 +29,11 @@ export interface INamespaceInvite {
 }
 
 export interface INamespaceSettings {
-  connection_announcement: string;
-  session_record: boolean;
+  connection_announcement?: string;
+  session_record?: boolean;
 }
 export interface INamespaceResponse {
-  settings: INamespaceSettings;
+  settings?: INamespaceSettings;
   id: string;
   tenant_id: string;
   name: string;

--- a/ui/src/store/api/namespaces.ts
+++ b/ui/src/store/api/namespaces.ts
@@ -18,8 +18,8 @@ export const leaveNamespace = async (tenant: string) => namespacesApi.leaveNames
 export const putNamespace = async (data: INamespaceResponse) => namespacesApi.editNamespace(data.id, {
   name: data.name,
   settings: {
-    connection_announcement: data.settings.connection_announcement,
-    session_record: data.settings.session_record,
+    connection_announcement: data.settings?.connection_announcement,
+    session_record: data.settings?.session_record,
   },
 });
 

